aspnet/fundamentals/logging.rst

```diff
@@ -162,7 +182,7 @@ Examining the Application event log in the Windows Event Viewer, the following e
 Configuring Other Providers
```

Comment:
The third party providers don't actually exist yet. We prototyped implementations to verify the design of the model, but these implementations don't ship anywhere. We are still working with the various logging providers to publish support for this logging model.